docs/Contributing.asciidoc

```diff
@@ -267,6 +267,17 @@ We use annotations in some places to mark "uncoverable" code such as this:
 
 See the docs for details https://metacpan.org/pod/Devel::Cover
 
+To run the tests with a different command than `prove` you can set the
+`PROVE` variable in the Makefile. For example, in our CircleCI tests we
+use `tools/prove_wrapper` to run the tests. This wrapper is a simple script that
+will run `prove` and fail the test if there is unhandled output. This is useful to
+avoid having unhandled output in tests. To run the tests with `tools/prove_wrapper` you can do:
```

Comment:
I suggest you configure your editor to consistently wrap at 80 characters. This line is too long

Reply:
I saw a lot more lines in the file that goes over the 80 character, does it still relevant in 2025?

Reply:
Given how big some people scale their fonts when sharing their screen, yes :)

Reply:
anyway I updated the doc accordingly